keyboards/cannonkeys/sagittarius/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Sagittarius",
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "width": 20.25,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
        "LAYOUT_default": {
```

Reply:
Ah dammit. I fixed the json to match the number of keys, but forgot to rename it to default again. Hoping that it passes linting.